cli/run.go

```diff
@@ -133,7 +138,13 @@ func (rc *RunCommand) runRun(args []string) error {
 
 	info, err := apiClient.ContainerGet(ctx, containerName)
 	if err != nil {
+		return err
+	}
 
+	if rc.rm {
+		if err := apiClient.ContainerRemove(ctx, containerName, true); err != nil {
+			return fmt.Errorf("failed to remove container %s: %v", containerName, err)
```

Comment:
If error happens at this point, shall we return an exit code of non-zero? 
With the change, we still return exit code of 0.

Reply:
As it a error happend here, this behaviour will get a error message, and of course, it should not be exit of 0, it will get exit code of 1 .

Reply:
sorry, I misunderstand your point.